public/locales/fr/common.json

```diff
@@ -54,25 +54,25 @@
   "lockArtist": "Verrouiller le champ de l'artiste",
   "lockAlbum": "Verrouiller le champ de l'album",
   "pasteIntercepted": "Le texte collé a été divisé automatiquement !",
-  "considerDonating": "S'il vous plait supportez le projet sur Patreon !",
+  "considerDonating": "Supportez le projet sur Patreon !",
   "footer": {
-    "joinCommunity": "Vous êtes le bienvenu à <1>rejoindre la communauté</1> sur Discord !",
-    "supportThisProject": "Veuillez envisager de <1>soutenir ce projet</1> sur Patreon :)",
-    "specialThanks": "Merci à tous les donateurs et les traducteurs ainsi qu'aux <1>contributeurs</1>"
+    "joinCommunity": "N'hésitez pas à <1>rejoindre la communauté</1> sur Discord, vous êtes le bienvenu !",
+    "supportThisProject": "N'hésitez pas à <1>soutenir ce projet</1> sur Patreon :)",
```

Comment:
These phrases were previously worked by @lapagen, I'd like to get their input as well. Personally I like the new translation for `joinCommunity`, but I'd also keep a slighlty different text in `supportThisProject` to avoid repeating "N'hésitez pas ..." - I know this is a nitpick so I'll go with whatever you both decide :slightly_smiling_face:

Reply:
I translated the sentences as I saw them from my point of view. We should check with @lapagen to see what he thinks.